### DISCUSSION TITLE
Using "Simultaneous-Use" Attribute in OpenWISP Radius Group Check to limit number of connections per user account

### DISCUSSION THREAD

**Discussion by 3pleo** [openwisp/openwisp-radius#564] [Q&A] [answered]
Hi,

I learnt that Freeradius supports using the "Simultaneous-Use" attribute to limit the number of simultaneous sessions (e.g. simultaneous connections on multiple devices) a particular account can have at the same time.  

- Is this supported on OpenWISP?
- Can I simply just add e.g. {Simultaneous-Use := 2}, group check attribute to a Radius Group?

Thanks.

Regards

**Comment by nemesifier** [comment]
I never tried this, please try and let us know what you find.
My assumption is that if freeradius supports it, OpenWISP shouldn't interefere with it, if that's not the case we should fix it.

**Comment by nemesifier** [comment, marked as answer]
The latest version of [OpenWISP RADIUS](https://github.com/openwisp/openwisp-radius/releases/tag/1.2.0) has built-in support for Simultaneous-Use. See https://openwisp.io/docs/stable/radius/user/simultaneous_use.html.
